tests/Functional/tests/render/interpreted-text/interpreted-text.rst

```diff
@@ -0,0 +1,6 @@
+Testing some `interpreted text`...
+
+`a` or `aa` or `.\/$!`
```

Comment:
Isn't code usually done with a double ```` in rst?  http://tpcg.io/_NZX3UJ

Reply:
Text in single backticks is considered "interpreted text" with the default "textrole". In Sphinx as well as in Githubs interpretation of RST, the default text-role is `<code>`. The only advantage of double backticks being, that you can use single backticks within, without escaping them. 

In RST-standard the default default-textrole is "title", a default which is less helpful in Documentation and is usually used in print. But each implementation may define its own fallback.